backend/tracim_backend/config.py

```diff
@@ -595,6 +540,21 @@ def __init__(self, settings: typing.Dict[str, typing.Any]):
                 'with a correct value'.format(self.FRONTEND_DIST_FOLDER_PATH)
             )
         self.load_ldap_settings(settings)
+        self._set_default_app(self.ENABLED_APP)
+
+        self.EMAIL_NOTIFICATION_NOTIFIED_CONTENTS = [
+            content_type_list.Page.slug,
+            content_type_list.Thread.slug,
+            content_type_list.File.slug,
+            content_type_list.Comment.slug,
+            # content_type_list.Folder.slug -- Folder is skipped
```

Comment:
This is legacy code i just move to avoid issue  (can't be place before set_default_app).  
We should probably remove this behaviour or change it anytime soon but not yet.